docs/03-tutorials/01-first-app.mdx

```diff
@@ -478,9 +542,15 @@ Widget build(BuildContext context) {
 }
 ```
 
-We trigger the note dialog when the action button is pressed and then save the note in the `onSaved` callback. To make the UI feel more responsive we add the changes to the notes list before calling `_createNote`. This way the note will be added to the list immediately and then updated when the server responds.
+Also import the dialog: 
```

Comment:
Could the import be incorporated into the code snipped above instead since it is essentially part of the same step?

Reply:
Yes, we can but since we are only providing the code snippet of build method here, I thought of writing the import statement separately. This import statement will help the user get rid of the lint.